filament/src/Froxelizer.cpp

```diff
@@ -588,12 +588,13 @@ void Froxelizer::froxelizeLoop(FEngine& engine,
         for (size_t i = offset; i < count; i += stride) {
             const size_t j = i + FScene::DIRECTIONAL_LIGHTS_COUNT;
             FLightManager::Instance li = instances[j];
-            LightParams light = {
-                    .position = (camera.view * float4{ spheres[j].xyz, 1 }).xyz, // to view-space
-                    .cosSqr = lcm.getCosOuterSquared(li),   // spot only
-                    .axis = vn * directions[j],             // spot only
-                    .invSin = lcm.getSinInverse(li),        // spot only
-                    .radius = spheres[j].w,
+			LightParams light;
+			{
```

Comment:
Why scope these assignments? You lost alignment on the comments too

Reply:
Same goes for every change related to structures intializations

Reply:
Scope just used for convenience. As for the indentation - it would be good to have clang-format file, which is now supported by Visual Studio, so everything will be formatted automatically.